pilot/pkg/serviceregistry/kube/controller/pod.go

```diff
@@ -29,6 +31,10 @@ import (
 	"istio.io/pkg/log"
 )
 
+const (
+	CNINetworkStatus = "k8s.v1.cni.cncf.io/networks-status"
```

Comment:
do not make kube controller parse all the nic ips. it is enough to get from the node meta.

Reply:
Already modified